doc/classes/CanvasItem.xml

```diff
@@ -640,7 +640,7 @@
 		</signal>
 		<signal name="item_rect_changed">
 			<description>
-				Emitted when the item rect has changed.
+				Emitted when the item rect's position, size or its [Sprite2D] texture has updated.
```

Comment:
That's not fully accurate as Sprite2D is not a texture of CanvasItem, it's a type of CanvasItem-derived node that has a texture property. I guess what you mean is that setting the texture of a Sprite2D will also trigger `CanvasItem.item_rect_changed`, which is correct. I'd suggest:

```suggestion
				Emitted when the item's boundaries (position and/or size) have changed, or when an action is taking that may or may not have impacted these boundaries (e.g. changing [member Sprite2D.texture]).
```

Reply:
Thanks, I've updated the commit.